tests/unit/games-dao.test.js

```diff
@@ -127,11 +127,11 @@ describe('Test games-dao', () => {
   });
 
 
-  it('', () => {
+  describe('Testing COUNT(1) related functions.', () => {
     const countOneFunctions = [
-      gamesDao.validateGame,
-      gamesDao.getGamesByMemberId,
-      gamesDao.isMemberInGame,
+      'validateGame',
+      'getGamesByMemberId',
+      'isMemberInGame',
     ];
     _.forEach(countOneFunctions, (fun) => {
       describe(`Test ${fun.name}`, () => {
```

Comment:
I believe this should just be `fun` instead of `${fun.name}` now that you changed `countOneFunctions`